.changeset/flat-worms-explode.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/heading': minor
```

Comment:
why did you switch to minors for this PR?

Reply:
Muscle memory

Reply:
👍 sticking to patches in mine just wanted to make sure

Reply:
I'll swap them to patches once I get back to this